### ISSUE TITLE
API key

### ISSUE THREAD

**Issue by hamelsveld** [lephisto/tesla-apiscraper#75]
Hi, 

now we need to provide the full username and password in de configfile .. can that be changed so that we only need the API key?

**Comment by lephisto** [comment]
Yes, that it possible. I just did not implement it because I'm lazy af..
